src/mutualinfo.jl

```diff
@@ -0,0 +1,40 @@
+# Mutual Information
+
+function _mutualinfo(A::AbstractMatrix{<:Integer})
+    N = sum(A)
+    (N == 0.0) && return 0.0
+
+    aᵢ = sum(A, dims=2)
```

Comment:
That looks cool, but I'm not sure it's immediately obvious from the var names that they refer to the cluster sizes of `a` and `b` from `mutualinfo(a, b)`.
FWIW, I'd keep it as it was in `vmeasure`.